### ISSUE TITLE
Accessibility and keyboard navigation

### ISSUE THREAD

**Issue by olach** [k-next/panel#57]
Related to #56

Apart from regular accessibility, being able to navigate the ui with the keyboard would be a good enhancement. 

This is a good resource with short to the point examples:
https://www.w3.org/TR/wai-aria-practices/examples/

More info:
https://www.w3.org/TR/wai-aria-practices/

**Comment by bastianallgeier** [comment]
I already added an issue to integrate accessibility tests: https://github.com/k-next/ui/issues/56 because I really think that we should have a11y at the core of everything we build. @fabianmichael made sure that all our new colors meet the WCAG 2.0 AA guidelines for contrast and I started introducing basic tab navigation accessibility for every navigation element in the latest commit: https://github.com/k-next/ui/commit/af85e147d930bce09e3503afb80eef4166e11e1d 

We are all no experts in this field, so any kind of help is very much appreciated!
